packages/babel-plugin-strip-runtime/src/__tests__/strip-runtime-source-code.test.ts

```diff
@@ -250,6 +251,53 @@ describe('babel-plugin-strip-runtime using source code', () => {
 
     describe('with jsx pragma', () => {
       it('work with classic jsx pragma', () => {
```

Comment:
nit: can we be specific on the behaviour we're observing? What does it work mean?

Reply:
ah hmm, it's more of a general check that `@compiled/babel-plugin`, `@compiled/babel-plugin-strip-runtime`, and the JSX-related plugins in [@babel/preset-react](https://babeljs.io/docs/babel-preset-react) all work together to create output that isn't invalid i suppose 🤔 

that said, i notice some things that i do want to specifically check for (e.g. jsx pragma is converted correctly), i'll see if i can make the snapshots more specific

incidentally, i've moved these JSX pragma-related tests into their own file `packages/babel-plugin-strip-runtime/src/__tests__/jsx-pragma.test.ts`, might as well include that in this PR since i'll be editing the tests anyway 😅